README.md

```diff
@@ -1,7 +1,54 @@
-# BlockScalars
+# MultilineStrings
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/BlockScalars.jl/stable)
-[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/BlockScalars.jl/dev)
-[![Build Status](https://travis-ci.com/invenia/BlockScalars.jl.svg?branch=master)](https://travis-ci.com/invenia/BlockScalars.jl)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/MultilineStrings.jl/stable)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/MultilineStrings.jl/dev)
+[![Build Status](https://travis-ci.com/invenia/MultilineStrings.jl.svg?branch=master)](https://travis-ci.com/invenia/MultilineStrings.jl)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+Tooling for manipulating multiline strings.
+
+## Features
+
+The package features the `@m_str` macro was inspired from [YAML's block scalars](https://yaml-multiline.info/) which provide options for manipulating multiline strings via a style and chomp indicator:
```

Comment:
Why are we going for just the letter `m` rather than something more descriptive?
Perhaps `multiline`?

Reply:
We can definitely rename this. As YAML calls these block scalars I was thinking of calling this `@block_str` but I was also thought as `@multiline_str` as an option. As the suffix is so terse already I was just kind of following that stlying.

Reply:
I think the way to think about this is "manipulation of a multiline string". As we have support for the "literal" option the result may still have multiple lines